### PR TITLE
BOM-1260

### PR DESCRIPTION
**Pull request by Ayub-Khan**
-Upgrade social-auth-app-django to support django22

### CODE REVIEW

File: requirements/all.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 #    make upgrade
 #
--e git+https://github.com/jsocol/django-ratelimit.git@72edbe8949fbf6699848e5847645a1998f121d46#egg=ratelimit
+-e git+https://github.com/jsocol/django-ratelimit.git@72edbe8949fbf6699848e5847645a1998f121d46#egg=django-ratelimit
```
Comment (jmbowman) [inline requirements/all.txt]: Why was this change needed?  We still have it the old way in edx-platform.

Comment (Ayub-Khan) [reply requirements/all.txt]: This fixes the warning 
```
WARNING:pip._internal.req.req_install:Generating metadata for package ratelimit produced metadata for project name django-ratelimit. Fix your #egg=ratelimit fragments.
```

Comment (jmbowman) [reply requirements/all.txt]: Interesting, I'll go re-read the docs on that `#egg` setting and see if we need to fix it elsewhere also.
